### ISSUE TITLE
Disable linters via source does not work in 0.30.0

### ISSUE THREAD

**Issue by mihai** [brigade/scss-lint#310]
Using the example from the [README](https://github.com/causes/scss-lint#disabling-linters-via-source) as `test.scss`:

``` scss
// scss-lint:disable BorderZero
p {
  border: none; // No lint reported
}
```

The linter complains instead of ignoring:

``` sh
test.scss:3 [W] BorderZero: `border: 0;` is preferred over `border: none;`
```
